src/components/Dashboard.js

```diff
@@ -1,4 +1,6 @@
 import React, { Component } from "react";
+import AddExpence from "./AddExpense";
```

Comment:
Nitpicking, but it should be named `AddExpense` as the filename 😄

Reply:
it is already.